src/android/InAppBrowser.java

```diff
@@ -814,23 +832,47 @@ private void sendUpdate(JSONObject obj, boolean keepCallback, PluginResult.Statu
     }
 
     /**
+     * The listener that gets called when InAppBrowserClient receives a message from JavascriptBridge
+     */
+    public interface WVJBResponseListener {
+        // or when data has been loaded
+        public void onDataReceived(Object data);
+    }
+
+    /**
      * The webview client receives notifications about appView
      */
-    public class InAppBrowserClient extends WebViewClient {
+    public class InAppBrowserClient extends WVJBWebViewClient {
         EditText edittext;
         CordovaWebView webView;
+        WVJBResponseListener wvjbResponseListener;
 
         /**
          * Constructor.
          *
          * @param webView
          * @param mEditText
          */
-        public InAppBrowserClient(CordovaWebView webView, EditText mEditText) {
-            this.webView = webView;
+        public InAppBrowserClient(WebView webView, CordovaWebView cordovaWebView, EditText mEditText) {
+            super(webView);
```

Comment:
yup, that's ok because its parent class WVJBWebViewClient doesn't call super at all.